patches/node/build_enable_perfetto.patch

```diff
@@ -290,7 +290,7 @@ index be0f55a409a71bf9c1763c36fdc252857228742e..827b5330b2f8c545338a46c548f8abf4
 +#endif
  
  // Adds a metadata event to the trace log. The |AppendValueAsTraceFormat| method
- // on the convertable value will be called at flush time.
+ // on the convertible value will be called at flush time.
```

Comment:
This typo belongs to Node's file `src/tracing/trace_event.h` that is being patched here. We need to leave the spelling alone in order for the patch to succeed.

(And in fact the typo goes further upstream than that; the file in Node's repo is copied from Chromium's `base/trace_event/trace_event.h` :smile_cat:)